lib/ansible/module_utils/crypto.py

```diff
@@ -15,11 +15,15 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 #
-# --------------------------------------------------------------
-# A clearly marked portion of this file is licensed under the BSD
+# ----------------------------------------------------------------------
+# A clearly marked portion of this file is licensed under the BSD license
 # Copyright (c) 2015, 2016 Paul Kehrer (@reaperhulk)
 # Copyright (c) 2017 Fraser Tweedale (@frasertweedale)
 # For more details, search for the function _obj2txt().
+# -----------------------------------------------------------------------------
+# A clearly marked portion of this file is licensed under the Apache License 2.0
```

Comment:
```suggestion
# A clearly marked portion of this file is extracted from a project that is licensed under the Apache License 2.0
```

Reply:
Done. I've added a line break to shorten this a bit (since the rest of the header is also not that long), I hope that's ok :)